src/components/UI/Header.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <h1 class="title">PMB PMK ITB 2021</h1>
+  <img class="logo" src="https://bit.ly/fcc-relaxing-cat" alt="Logo PMB PMK ITB 2021" />
```

Comment:
Hal yang harus diperhatikan:
- Tulisan title dan logo-nya tidak tepat di tengah axis `x`. 
- Logo dan title-nya juga masih saling menimpa satu sama lain.

To do:
- Gunakan flexbox agar mereka bisa rapi.
- Coba jangan gunakan `position: absolute` dan `px` sebagai measurement. Tapi gunakan seperti `rem`.
- (Optional) Coba gunakan Tailwind CSS untuk mengatur title dan logo-nya. Tailwind sudah terinstall di project, bisa langsung pakai. Contoh penggunaan Tailwind: [FloatingBox.vue](https://github.com/pmk-itb/form-registrasi-pmb/blob/feat/thank-you-box/src/components/UI/FloatingBox.vue) dan [ThankYou.vue](https://github.com/pmk-itb/form-registrasi-pmb/blob/feat/thank-you-box/src/views/ThankYou.vue)

![title-and-logo](https://i.imgur.com/2jqUUiM.png)

Referensi:
- [Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)
- [Tailwind](https://tailwindcss.com/docs)